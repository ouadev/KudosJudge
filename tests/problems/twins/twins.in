21
11 99 88 17 19 20 12 13 33 44
11 12 13 14 15 16 66 88 19 20
20 18 55 66 77 88 17 33 44 11
12 23 34 45 56 67 78 89 91 18
17 11 22 33 44 55 66 77 88 99
11 17 22 33 44 55 66 77 88 99
11 22 33 44 55 66 77 88 17 99
11 22 33 44 55 66 77 88 99 17
11 22 18 33 44 55 66 77 88 99
11 22 33 44 55 66 77 18 88 99
11 22 33 18 44 55 66 77 88 99
11 22 33 44 55 66 18 77 88 99
18 17 20 30 40 50 60 70 80 90
20 30 40 50 60 70 80 90 17 18
17 20 30 40 50 60 70 80 90 18
20 18 30 40 50 60 70 80 17 90
20 30 17 18 40 50 60 70 80 90
20 30 40 50 18 17 60 70 80 90
27 37 47 57 67 77 87 97 98 99
28 38 48 58 68 78 88 98 97 99
35 20 30 40 50 60 70 80 90 55
