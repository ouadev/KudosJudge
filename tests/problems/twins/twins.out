11 99 88 17 19 20 12 13 33 44
zack

11 12 13 14 15 16 66 88 19 20
none

20 18 55 66 77 88 17 33 44 11
both

12 23 34 45 56 67 78 89 91 18
mack

17 11 22 33 44 55 66 77 88 99
zack

11 17 22 33 44 55 66 77 88 99
zack

11 22 33 44 55 66 77 88 17 99
zack

11 22 33 44 55 66 77 88 99 17
zack

11 22 18 33 44 55 66 77 88 99
mack

11 22 33 44 55 66 77 18 88 99
mack

11 22 33 18 44 55 66 77 88 99
mack

11 22 33 44 55 66 18 77 88 99
mack

18 17 20 30 40 50 60 70 80 90
both

20 30 40 50 60 70 80 90 17 18
both

17 20 30 40 50 60 70 80 90 18
both

20 18 30 40 50 60 70 80 17 90
both

20 30 17 18 40 50 60 70 80 90
both

20 30 40 50 18 17 60 70 80 90
both

27 37 47 57 67 77 87 97 98 99
none

28 38 48 58 68 78 88 98 97 99
none

35 20 30 40 50 60 70 80 90 55
none
